packages/strapi-design-system/src/v2/SimpleMenu/SimpleMenu.js

```diff
@@ -20,7 +21,12 @@ const OptionButton = styled.button`
   ${getOptionStyle}
 `;
 
-const OptionLink = styled(BaseLink)`
+const OptionLink = styled(NavLink)`
```

Comment:
we should use `BaseLink` instead of `NavLink` here

Reply:
Noted.